scene/gui/rich_text_label.h

```diff
@@ -354,20 +353,12 @@ class RichTextLabel : public Control {
 	};
 
 	struct ItemCustomFX : public ItemFX {
-		Ref<CharFXTransform> char_fx_transform;
-		Ref<RichTextEffect> custom_effect;
+		Ref<class CharFXTransform> char_fx_transform;
+		Ref<class RichTextEffect> custom_effect;
```

Comment:
Forward declarations should be created at the top of the file. (although it's interesting that you can inline them 🤔)